server/src/lexer/pest/python.pest

```diff
@@ -82,7 +82,10 @@ block_comment_ending = { newline_eoi }
 
 // ## Code blocks
 code_block = { code_line+ }
-code_line = _{ (!doc_block ~ logical_line ~ NEWLINE) | (!doc_block ~ not_newline+ ~ EOI) }
+single_line_string = {("\"" ~ (!("\"") ~ ANY)* ~ "\"") | ("'" ~ (!("'") ~ ANY)* ~ "'")}
+// If there is text or characters within the triple quoted string, it will be regarded as a code block.
+multi_line_string = { ("'''" ~ (!("'''") ~ ANY)* ~ "'''") | "\"\"\"" ~ (!("\"\"\"") ~ ANY)* ~ "\"\"\""}
+code_line = _{ (!single_line_string ~ NEWLINE | not_newline) | (multi_line_string ~ NEWLINE) | !(inline_comment_delim_0) ~ ("'''" | "\"\"\"") | (!doc_block ~ not_newline+ ~ EOI) }
```

Comment:
Not quite right. I think you want to say that code: not a comment, but is a single-line or multi-line string. Here, you say it's not a single/multi-line string.

Reply:
Yes, I was trying out some test cases and forgot to update the code since the single-line string was the only one recognized.